modules/ROOT/partials/configuration/advanced-typography.adoc

```diff
@@ -0,0 +1,202 @@
+[[content_langs]]
+== `+content_langs+`
+
+This option sets which languages are available for applying language-specific typography rules.
+
+A user can then select text and select from the available languages. That language’s language-specific typography rules are then applied to the selected text.
+
+*Type:* `+Object+`
+
+=== Example: Using `content_langs`
+
+[source,js,subs="attributes+"]
+----
+tinymce.init({
+  selector: 'textarea', // change this value according to your html
+  plugins: '{plugincode}',
+  toolbar: '{plugincode} language',
+  language: 'en_US', // This is required to configure the default language used by typography.
+  content_langs: [
+    { title: 'English', code: 'en' },
+    { title: 'Spanish', code: 'es' },
+    { title: 'German', code: 'de' }
+    ],
+});
+----
+
+=== Currently supported languages
+
+[cols="1,1"]
+|===
+|Supported language |Language code 
+
+|Belarusian
+|be
+
+|Bulgarian
+|bg
+
+|Catalan
+|ca
+
+|Danish
+|da
+
+|Dutch
+|nl
+
+|English (UK)
+|en-GB
+
+|English (US)
+|en-US
+
+|Esperanto
+|eo
+
+|Estonian
+|et
+
+|Finnish
+|fi
+
+|German
+|de
+
+|Greek
+|el
+
+|Hungarian
+|hu
+
+|Irish
+|ga
+
+|Italian
+|it
+
+|Latvian
+|lv
+
+|Norwegian
+|no
+
+|Polish
+|pl
+
+|Romanian
+|ro
+
+|Russian
+|ru
+
+|Servian
+|sr
+
+|Slovak
+|sk
+
+|Slovenian
+|sl
+
+|Spanish
+|es
+
+|Swedish
+|sv
+
+|Turkish
+|tr
+
+|Ukraninan
+|uk
+
+|===
+
+[[typography_custom_rules]]
+== `+typography_custom_rules+`
```

Comment:
Do we need to make it clear how these interact with other built-in rules? If they are given the same name, do they clobber the originals?

Reply:
`typography_custom_rules` feature was not implemented in this release.

Reply:
@yacodes,

> `typography_custom_rules` feature was not implemented in this release.

Adding this comment out of an abundance of caution.

I’m parsing the above to mean *the `typography_custom_rules` option is not part of the current Advanced Typography plugin release and, therefore, does not currently need to be documented.*

Given that reading, I quote from the commit message:

> Placed `typography_custom_rules` section inside a comment block. As per @yacode’s note that
>
>> `typography_custom_rules` feature was not implemented in this release.